### PR TITLE
GLT-544 add dialogs to notify users that they have stumbled across unfinished features

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

**Comment by apmasell** [comment]
:+1: Bravo on the Nick Cage.

File: miso-web/src/main/webapp/scripts/library_ajax.js [hunk 1]
```diff
@@ -1051,7 +1051,18 @@ Library.ui = {
       alert("Please select one or more Libraries to dilute.");
       return false;
     }
-    alert("Finish methods to actually make the table and page.");
+    var cageDiv = '<div id="cageDialog"><span class="dialog">Look for this feature in the next release!<br>' 
+      + '<img src="http://images.mentalfloss.com/sites/default/files/styles/insert_main_wide_image/public/uncanny_valley.jpg"/></span></div>';
+    document.getElementById('go').insertAdjacentHTML('afterend', cageDiv);
```
Comment (tonydebat) [inline miso-web/src/main/webapp/scripts/library_ajax.js]: This line throws a TypeError in the javascript console. `document.getElementById(...) is null`

**Comment by tonydebat** [comment]
:+1: After error above corrected.

File: miso-web/src/main/webapp/scripts/library_ajax.js (hunk 1 above)
Comment (callunity) [inline miso-web/src/main/webapp/scripts/library_ajax.js]: Aha, had forgotten to add in the save button. That's fixed now.

**Pull request merged**